src/mnemonics/electrum-words.cpp

```diff
@@ -436,6 +436,15 @@ namespace crypto
       return bytes_to_words(src.data, sizeof(src), words, language_name);
     }
 
+    std::string_view bytes_to_words(const crypto::secret_key& src,
+      const std::string &language_name)
+    {
+      epee::wipeable_string epee_words;
+      if (!bytes_to_words(src.data, sizeof(src), epee_words, language_name))
+        throw std::runtime_error("Failed to create seed from key for language: " + language_name);
+      return epee_words.view(); 
+    }
+
```

Comment:
this function doesn't make sense.

it is returning a string_view of a wipeable_string that is local to the function. i dont think it'll work. if it does, prepare the holy water.

Reply:
Yeah changed this to return a std::string, although we lose the wiping of the data now. So we might want to implement wipeable string ourselves to get out of epee land